plugins/woocommerce/client/blocks/assets/js/blocks/product-details/block.json

```diff
@@ -12,6 +12,15 @@
 		},
 		"align": [ "wide", "full" ]
 	},
-	"attributes": {},
+	"attributes": {
+		"align": {
+			"type": "string",
+			"default": "wide"
```

Comment:
I see default value wasn't assigned in the PRR. I don't know if it's good, bad or neutral, but mentioning to make sure it's expected.

EDIT: I saw a comment in `migrate` function so I assume it's ok. Leaving the comment to document thought process 😄